### DISCUSSION TITLE
Get Tutorial 'End-to-End Adopter Journey' Running

### DISCUSSION THREAD

**Discussion by tischroe** [eclipse-tractusx/eclipse-tractusx.github.io#782] [Q&A]
Following the instructions of the ['End-to-End Adopter Journey'-tutorial](https://eclipse-tractusx.github.io/docs/tutorials/e2e) locally on Ubuntu 22.04.4, I get the follwing warnings and errors after executing `terraform apply`:

![Screenshot from 2024-03-22 10-49-53](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/141323482/c9238da9-d6f4-4128-a79b-1ef543d01673)

Are there any ideas if this is likely a problem with my personal setup or are there known issues with the tutorial that could be linked to it?

**Comment by stephanbcbauer** [comment]
Hello @tischroe,

Thank you very much for your discussion. There is already a short workaround which is not implemented yet, but the change is described [here](https://github.com/eclipse-tractusx/tutorial-resources/issues/257).

In a "nutshell" :) change in the main.tf

<img width="1169" alt="image" src="https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/84396022/4d7688d0-c86c-4b70-88ff-7f9bfef84d97">

Please reach out to me, if this worked

Stephan
